### ISSUE TITLE
Store and Record classes are no longer recognized

### ISSUE THREAD

**Issue by dawnarius** [tekartik/sembast.dart#144]
Hello,
this code used to work until early January and I didn't work on the project since then
but now with the updated version of dart / flutter It wont work anymore and Store and Record classes are no longer recognized
Also the Database class doesn't implements a putRecord method anymore

```
  final Store _store;
  final Database _db;

  @override
  Future<User> createUser(
      String username, String password, String emailAddress) async {
    final User user = User(username, password, emailAddress);
    final Map<String, dynamic> values = convertItemToStorageMap(user);
    final Record recordToAdd = Record(_store, values, user.objectId);
    final Record recordFromDB = await _db.putRecord(recordToAdd);
    return convertRecordToItem(record: recordFromDB);
  }
```

**Comment by alextekartik** [comment]
This API has been deprecated since version 2 and new API is available since version 1.5 (June 2019). See doc here https://github.com/tekartik/sembast.dart/blob/master/sembast/doc/new_api.md. You can limit the sembast version to 2.2 to make your code still work and upgrade it to the new API

**Comment by dawnarius** [comment]
Thank you very much
